TicTacToe.Tests/GameTests.cs

```diff
@@ -14,10 +14,10 @@ public class GameTests
         public void CanCreateNewGame() => Assert.IsType<Game>(game);
 
         [Fact]
-        public void GameHasABoard() => Assert.IsType<Char[]>(game.Board);
+        public void GameHasABoard() => Assert.IsType<Board>(game.Board);
 
-        [Fact]
-        public void GameBoardHasNineSpaces() => Assert.Equal(9, game.Board.Length);
+        // [Fact]
+        // public void GameBoardHasNineSpaces() => Assert.Equal(9, game.Board.Length);
```

Comment:
If something is not needed, let's try to remove it instead of comment it out. That way it'll help keep things looking clean.